### DISCUSSION TITLE
Training with partially annotated images

### DISCUSSION THREAD

**Discussion by maldivien** [AlexeyAB/darknet#7797] [Q&A] [answered]
I have dataset of 15 classes and some images are duplicated with different class names to annotate separately.

Example,

Img1.jpg - - - has a person and car but only car is annotated

img1.jpg (renamed-same image as above ) - - - only car is annoated

My question how does this effect the training, does yolo consider background as negatives?

What's the easy way around this solution, combine all annotations per image?

**Comment by stephanecharette** [comment, marked as answer]
Your question is:  2 identical images, but only the car is annotated in the first image, and only the car (again?) is annotated in the duplicate 2nd image.  Did you make a mistake?

If you mean that you only annotate the car in the first image, and only the person in the 2nd image, it will not work.  You'll destroy the training.

See https://www.ccoderun.ca/programming/darknet_faq/#image_markup and https://www.ccoderun.ca/darkmark/ImageMarkup.html

**Reply by maldivien** [reply]
Thanks 👍 Is there an easy way to know during the training that the network is overfitting.
